aptos-move/framework/aptos-token/sources/token_event_utils.move

```diff
@@ -0,0 +1,318 @@
+/// This module provide utils to create and emit token events
+module aptos_token::token_event_utils {
```

Comment:
nit: May be just name this token events v2?

Reply:
rename to token_event_store. we won't have v3, ..... . If we need to revamp, we can add new resource  TokenEventStoreV2 in this module to accommodate new events

Reply:
Makes sense